python/lsst/ip/isr/deferredCharge.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, size, emission_time, pixel, trap_type, coeffs):
             raise ValueError('Unknown trap type: %s', self.trap_type)
 
         if self.trap_type == 'spline':
-            centers, values = np.split(np.array(self.coeffs, dtype=np.float), 2)
+            centers, values = np.split(np.array(self.coeffs, dtype=np.float64), 2)
```

Comment:
I think everything that comes from a calibration should be `float64`, so I'm happy with this fix.